WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -89,11 +89,10 @@ final class ProductFormViewController: UIViewController {
     private var cancellable: ObservationToken?
 
     init(product: Product,
-         currency: String,
          presentationStyle: PresentationStyle,
          isEditProductsRelease2Enabled: Bool,
          isEditProductsRelease3Enabled: Bool) {
-        self.currency = currency
+        self.currency = CurrencySettings.shared.symbol(from: CurrencySettings.shared.currencyCode)
```

Comment:
nit: how about we DI the `CurrencySettings` with default to `CurrencySettings.shared` and initialize the `currency` here with the `CurrencySettings` instance, so that we can unit test this in the future? 🙂

Reply:
👍 sound good. c4c4f02d4fc7cba58874184864444e8d940cb470